### DISCUSSION TITLE
started an automated service failed

### DISCUSSION THREAD

**Discussion by AIWildUItraman** [ifm/ifm3d#411] [Q&A]
HI @theseankelly @tpanzarella 
I started an automated service following this [document](https://ifm3d.com/documentation/SoftwareInterfaces/Docker/autostart.html), which involves launching a Docker container and then running an image acquisition program inside the Docker container. However, every time I boot up, this service fails to start correctly due to the startup sequence. I am not sure which program should precede the service program to ensure the image acquisition works properly once inside Docker. Could you please assist me with this?

**Comment by graugans** [comment]
Hej @AIWildUItraman,

neither Tom nor Sean work for ifm any longer, we are happy to assist you if you could please be so kind and provide us more details. For example how you are trying to approach this and what the error message looks like.
